### PR TITLE
Blog post situating the project

### PR DESCRIPTION
**Pull request by ghost**
New blog post, comments are welcome

### CODE REVIEW

File: _posts/2022-02-25-situating-the-project.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+
+## Registering Changes in how People make Sense of Times of Transition
+Eventually, the point of the project is not simply to create programs for computational analysis. The point is rather to be able to analyze stories in times of transition, and to analyze *more* of them. This is a social and political goal: We want to register changes in how people make sense of times of transitions. What did (does?) Covid-19 really mean for people in precarious situations? What do political measures against Covid-19 mean for those with chronic illnesses, and what does the reversal of those measures mean for them? As Anneke Sools^5 asked in her research: How do people narrate their futures in uncertain times, and how can this knowledge improve our world? In collaboration with other organizations, we have access to (open) databases of these personal stories, which we can use to train and test our software. All the while staying close to the texts and what's critical in these stories.
```
Comment (eriktks) [inline _posts/2022-02-25-situating-the-project.md]:
```suggestion
Eventually, the point of the project is not simply to create programs for computational analysis. The point is rather to be able to analyze stories in times of transition, and to analyze *more* of them. This is a social and political goal: We want to register changes in how people make sense of times of transitions. What did (does?) Covid-19 really mean for people in precarious situations? What do political measures against Covid-19 mean for those with chronic illnesses, and what does the reversal of those measures mean for them? As Anneke Sools[^5] asked in her research: How do people narrate their futures in uncertain times, and how can this knowledge improve our world? In collaboration with other organizations, we have access to (open) databases of these personal stories, which we can use to train and test our software. All the while staying close to the texts and what's critical in these stories.
```

**Review by eriktks — APPROVED** [review]
Looks good. One error: at reference 5. And perhaps one or two images could be added? (see post 1) Thanks for this post!

File: _posts/2022-02-25-situating-the-project.md (hunk 1 above)
Comment (ghost) [reply _posts/2022-02-25-situating-the-project.md]: Thanks Erik, I will fix the reference and add an image.

**Comment by eriktks** [comment]
Post has already been added

**Pull request closed without merging**